ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -304,6 +308,39 @@ - (void)getAccessTokenWithScope:(NSString* _Nullable)scope forceRefreshing:(BOOL
 
 }
 
+- (void)refreshTokenForScope:(NSString* _Nonnull)scope callback:(IDmeVerifyWebVerifyTokenResults _Nonnull)callback {
```

Comment:
`getAccessTokenWithScope` accepts a nullable scope value. Shouldn't this be similar to be consistent?